c/include/libsbp/ssr.h

```diff
@@ -114,8 +114,11 @@ but in units of TECU instead of m.
  * Troposphere vertical delays at the grid point.
  */
 typedef struct SBP_ATTR_PACKED {
-  s16 hydro;    /**< Hydrostatic vertical delay [4 mm (add 2.3 m to get actual vertical hydro delay)] */
-  s8 wet;      /**< Wet vertical delay [4 mm (add 0.252 m to get actual vertical wet delay)] */
+  s16 hydro;     /**< Hydrostatic vertical delay [4 mm (add 2.3 m to get actual vertical hydro delay)] */
```

Comment:
why did an extra space get added here?

Reply:
The generator inserts enough spaces so that there's 4 spaces between the longest variable and the comments, then aligns the remaining comments to that. https://github.com/swift-nav/libsbp/blob/791bf79fa092fe160a56b089dab9266a6417407c/generator/sbpg/targets/resources/sbp_messages_template.h#L48